mappings/net/minecraft/predicate/entity/EntityPredicate.mapping

```diff
@@ -75,3 +78,38 @@ CLASS net/minecraft/class_2048 net/minecraft/predicate/entity/EntityPredicate
 			ARG 1 effects
 		METHOD method_8924 distance (Lnet/minecraft/class_2025;)Lnet/minecraft/class_2048$class_2049;
 			ARG 1 distance
+	CLASS class_5258 Extended
+		FIELD field_24388 EMPTY Lnet/minecraft/class_2048$class_5258;
+		FIELD field_24389 conditions [Lnet/minecraft/class_4570;
+		FIELD field_24390 combinedCondition Ljava/util/function/Predicate;
+		METHOD <init> ([Lnet/minecraft/class_4570;)V
+			ARG 1 conditions
+		METHOD method_27803 ofLegacy (Lnet/minecraft/class_2048;)Lnet/minecraft/class_2048$class_5258;
+			ARG 0 predicate
+		METHOD method_27804 toJson (Lnet/minecraft/class_5267;)Lcom/google/gson/JsonElement;
+			ARG 1 saver
```

Comment:
Shouldn't `saver` be `serializer`? Saver sounds like it's writing the json to the file or something, but the method returns a JsonElement.

Reply:
the saver is an object that has a method that converts a list of loot conditions to a json element. 
the saver is a singleton, and i am not very sure why mojang even made the saver object, as the only used method is final and only a singleton is ever passed; the used method is only used in one place as well

Reply:
> converts a list of loot conditions to a json element

> the saver is a singleton

That sounds exactly like a serializer...

Reply:
so loader/saver to deserializer/serializer?